draft-huitema-dnssd-privacyscaling.xml

```diff
@@ -532,8 +588,135 @@ This draft results from initial feedback in the DNS SD working group on <xref ta
   <seriesInfo name="DOI" value="10.1109/HPCC.2014.141"/>
 </reference>
 
+<reference anchor="Wu16" target="https://arxiv.org/pdf/1604.06959.pdf%22">
+  <front>
+    <title>Privacy, discovery, and authentication for the internet of things</title>
+    <author initials='D.' surname='Wu' fullname='David Wu'>
+      <organization>Stanford University</organization>
+    </author>
+    <author initials='A.' surname='Taly' fullname='Ankur Taly'>
+      <organization>Google</organization>
+    </author>
+    <author initials='A.' surname='Shankar' fullname='Asim Shankar'>
+      <organization>Google</organization>
+    </author>
+    <author initials='D.' surname='Boneh' fullname='Dan Boneh'>
+      <organization>Stanford University</organization>
+    </author>
+    <date year="2016"/>
+  </front>
+</reference>
+
+<reference anchor="SIGMA" target="http://link.springer.com/content/pdf/10.1007/978-3-540-45146-4_24.pdf">
+  <front>
+    <title>SIGMA: The 'SIGn-and-MAc'approach to authenticated Diffie-Hellman and its use in the IKE protocols</title>
+    <author initials='H.' surname='Krawczyk' fullname='Hugo Krawczyk'>
+      <organization>EE Department, Technion, Haifa, Israel, and IBM T.J. Watson Research Center</organization>
+    </author>
+    <date year="2003"/>
+  </front>
+</reference>
+
 
 </references>  
 
+<section title="Survey of Implementations">
+
```

Comment:
Let's keep that text for now, but I am concerned that this is not the right way to approach the issue. The "private IOT" by Boneh & al. does not describe an actual standard. It is basically a research paper that could be the basis for a future system. As such, we could cite it as a reference when describing scaling solutions -- I would document it as the "broadcast encryption" approach.

Similarly, this draft is not the right place to comment on the dnssd privacy draft, which is a working group document. Such comments really belong on the mailing list, expressing concerns for example about scaling properties. The normal process is that these concerns will lead to a revision of the draft, including possibly big changes in the way secrets are managed. But then, if the draft does change, the survey section will become obsolete.

Reply:
@huitema The survey should be living text. Perhaps this is not the right place for it. However, I do believe it should exist, as it greatly influences (a) scaling issues, (b) privacy requirements, and (c) proposed, standard solutions. Maybe we should just move this to a separate document? Would that suffice?